### PR TITLE
Kjq/awsclient

### PR DESCRIPTION
**Pull request by kentquirk**
This is a modified highwater client that uses AWS DynamoDB as well as KISSMetrics to store log entries. I'd like to deploy it, compare logs for a few days, and then turn off KM.

### CODE REVIEW

**Comment by kentquirk** [comment]
@jh-bate Wanna take a look, please?

File: lib/dynamo/dynamoDbClient.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        _n: eventName
+      };
+      // do the timestamp if it's specified
+      var args = _.clone(event);
+      log.info('metric[%s], user[%s]: %j', parms._n, parms._p, event);
+
+      if (args.timestamp) {
+        parms._t = args.timestamp;
+      } else {
+        parms._t = Date.now();
```
Comment (jh-bate) [inline lib/dynamo/dynamoDbClient.js]: or should it be `=  new Date().toISOString()`?

**Comment by jh-bate** [comment]
@kentquirk LGTM. Idea of deploying and comparing the logs sounds solid to me

File: lib/dynamo/dynamoDbClient.js (hunk 1 above)
Comment (kentquirk) [inline lib/dynamo/dynamoDbClient.js]: No, actually -- date needs to be an integer value in this case, for compatibility with the KISSMetrics logs.

**Pull request merged**